cfme/configure/access_control.py

```diff
@@ -1237,6 +1243,10 @@ def is_displayed(self):
 class DetailsTenantView(ConfigurationView):
     """ Details Tenant View """
     toolbar = View.nested(AccessControlToolbar)
+    name = Text('Name')
+    description = Text('Description')
+    parent = Text('Parent')
+    list_entity = ListEntity
```

Comment:
**REQURIED**
Where is `list_entity` used? Also you should initilaize it.

Reply:
Initialized